content/docs/examples/multicluster/gke/index.md

```diff
@@ -148,9 +147,7 @@ $ kubectl get pods -n istio-system
       --name istio-remote \
       --set global.remotePilotAddress=${PILOT_POD_IP} \
```

Comment:
for followup work, you also shouldn't need `remotePolicyAddress` nor `remoteTelemetryAddress`, although you would of course want to test that :)

Reply:
By follow up work, you mean the single gateway load balancer IP to expose the telemetry, Pilot service?